source/routing/raptor_api.cpp

```diff
@@ -206,6 +206,22 @@ static void co2_emission_aggregator(pbnavitia::Journey* pb_journey){
     }
 }
 
+static void calculate_total_walking_duration(pbnavitia::Journey* pb_journey) {
```

Comment:
Please rename it `compute_durations` as the fields in the durations object may change later.